### DISCUSSION TITLE
Missing installation options (KE)

### DISCUSSION THREAD

**Discussion by crypticvoyage** [Guilouz/Creality-Helper-Script-Wiki#600] [Issues]
Hi All,

I have a Ender 3 KE and have installed a fresh version of the helper script but seem to have some options missing. I've updated to the latest firmware on the printer as well. Namely options such as the useful macros and KAMP is missing. I contemplated manually running the script in the /usr/data/helper-script/scripts for the macros but wasn't sure if it would break anything.

<img width="458" alt="image" src="https://github.com/Guilouz/Creality-Helper-Script-Wiki/assets/153978536/cdc1f4a1-b6b4-4e5f-9f20-e0ae57c125df">

**Comment by bferrarese** [comment]
Hi there,

Did you ever figure it out? Mine looks exactly like that.
